File: source-connector/src/main/java/sourceconnector/SourceConnector.java
package sourceconnector;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.boot.context.properties.ConfigurationPropertiesScan;

@SpringBootApplication
@ConfigurationPropertiesScan
public class SourceConnector {
  public static void main(String[] args) {
    SpringApplication.run(SourceConnector.class, args);
  }
}

File: source-connector/src/main/kotlin/sourceconnector/SourceConnector.kt
package sourceconnector

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.context.properties.ConfigurationPropertiesScan
import org.springframework.boot.runApplication

@SpringBootApplication
@ConfigurationPropertiesScan
class SourceConnector

fun main(args: Array<String>) {
  runApplication<SourceConnector>(*args)
}
